plugins/restapi/handlers.go

```diff
@@ -315,6 +326,123 @@ func (p *Plugin) registerHTTPHandler(key, method string, f func() (interface{},
 	p.HTTPHandlers.RegisterHTTPHandler(key, handlerFunc, method)
 }
 
+// jsonSchemaHandler returns JSON schema of VPP-Agent configuration.
+func (p *Plugin) jsonSchemaHandler(formatter *render.Render) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		// create FileDescriptorProto for dynamic Config holding all VPP-Agent configuration
+		knownModels, err := client.LocalClient.KnownModels("config") // locally registered models
+		if err != nil {
+			errMsg := fmt.Sprintf("500 Internal server error: "+
+				"can't get registered models: %v\n", err)
```

Comment:
looks a bit strange to duplicate HTTP status code and its text in the response body. I think, you may omit that `"500 Internal server error: "` or is it required? :thinking:

Reply:
It was copy/paste of error handling from other rest handlers, but i can omit that. It looks strange and the user of my REST API addition don't need that. However, don't know about other handlers. I guess it is only good for logging and not for response (logging and response body use the same variable). So i preserved it for logging and left out from http response message body.
Fixed in [`cd09327`](https://github.com/ligato/vpp-agent/pull/1784/commits/cd093279785a4b0ce46f06cc6b03e9d21bd12a76).